### DISCUSSION TITLE
Add Tag Count to Edit Mass Tags

### DISCUSSION THREAD

**Discussion by Vyeche** [publishpress/publishpress-taxonomies#1521] [Feature Requests]
Hi

Added Tag Count to Edit Mass Tags in my fork. Let me know if it's useful? I don't know how to make it sort, but its a start.

https://github.com/Vyeche/simple-tags

**Comment by herewithme** [comment]
Order by tag count is not easy :
https://wordpress.stackexchange.com/questions/249202/order-by-most-used-tag

**Comment by herewithme** [comment]
Start branch here :
https://github.com/BeAPI/simple-tags/tree/issue/93
